docs/source/tutorial.rst

```diff
@@ -363,9 +367,10 @@ Genome binning
 **************
 
 The following step is to generate bins from the previous draft genomes or
-contigs (wether from MetaSPAdes or MEGAHIT). To do so, we will use three
+contigs. To do so, we will use the contigs from MetaSPAdes's output through three
```

Comment:
I think it is good to keep the `(wether from MetaSPAdes or MEGAHIT)` part, but stating that in this tutorial the results from **MEGAHIT** will be used (If we say MetaSPAdes then we would have to change the following parts of the tutorial). It would be also necessary to change **wether** for **whether** or use **either** instead.
```suggestion
contigs (either from MetaSPAdes or MEGAHIT). In this tutorial, we will
use the contigs from MEGAHIT's output through three
```